docs/LEANCLOUD-COUNTER-SECURITY.md

```diff
@@ -1,3 +1,9 @@
+# DEPRECATED!!
```

Comment:
How about add the warnings here: https://github.com/theme-next/hexo-leancloud-counter-security ?

Reply:
will do when this merged